### ISSUE TITLE
bvh to vmd converter

### ISSUE THREAD

**Issue by nikeyb** [sugiany/blender_mmd_tools#59]
Could you make a converter bvh to vmd?

**Comment by Hogarth-MMD** [comment]
Take a look here for information and discussion of this subject:
https://github.com/powroupi/blender_mmd_tools/issues/60

The forked version of mmd_tools by powroupi is now much more developed and superior to sugiany's original version of mmd_tools. It can import and export both .vmd and .vpd.

**Comment by Hogarth-MMD** [comment]
@powroupi is the current developer of mmd_tools.
You can find lots of great bug fixes and additional features in the version of mmd_tools by @powroupi:
https://github.com/powroupi/blender_mmd_tools/archive/dev_test.zip
https://github.com/powroupi/blender_mmd_tools
https://github.com/powroupi/blender_mmd_tools/issues

**Comment by ElesonTan84** [comment]
Can you make some sort of visual tutorial for converting bvh to vmd because i have almost no idea as some of the instrustions is making me confuse like renaming bones but i have no idea where to rename them in blender

**Comment by Hogarth-MMD** [comment]
I made a tutorial about converting bvh to vmd and various other tutorials, which you can find linked here in the mmd_tools wiki (powroupi fork):
https://github.com/powroupi/blender_mmd_tools/wiki/Links-to-mmd_tools-Tutorials

Sorry, I haven't gotten around to adding images to the BVH to VMD tutorial yet. My mmd_tools_helper add-on can be used to do a mass renaming of bones in Blender.

**Comment by ElesonTan84** [comment]
ok thx but what about the ik rig or even the vertex?

**Comment by ElesonTan84** [comment]
does the mass renamer helper can rename any bones like from PrioVr Mocap Studio?

**Comment by Hogarth-MMD** [comment]
No, if you download it, you can see the list of supported bone maps.

**Comment by Hogarth-MMD** [comment]
To correct myself:

The 3DS Max bone map is supported by mmd_tools_helper, so I assume that the mmd_tools_helper bones renamer operator should work with Yost Labs software, such as PrioVR or Yost Labs MoCap Studio (both open source).

https://yostlabs.com/yost-labs-mocap-studio/

"Below is a collection of sample BVH animation exports from the Yost Labs 3-Space Mocap Studio. They are compatible with 3ds Max biped naming scheme.

Capture session from the Quick Start video, updated 25 Nov 2013
Actor throwing a football, updated 25 Nov 2013
Actor catching and throwing a basketball, updated 25 Nov 2013
Actor walking, updated 25 Nov 2013
Actor performing actions with a sword and shield, updated 25 Nov 2013
Actor performing a zombie walk, updated 25 Nov 2013
Actor performing a zombie attack, updated 25 Nov 2013"

https://yostlabs.com/support/priovr-downloads/
The Yost Labs PrioVR Utility is an open source application for setup and configuration or the PrioVR Dev Kit.

**Comment by ElesonTan84** [comment]
Ok. Thanks for letting me know about this.

**Comment by ElesonTan84** [comment]
Why is my character not posing? i meant like it is moving but not the arms and leg are not moving like in the animation. 

https://www.youtube.com/watch?v=Amrw0m-ozDM
